content/4.examples/send-payment.md

```diff
@@ -0,0 +1,87 @@
+# Send Payment
+
+This example shows how to send a payment from one account to another via Gevamu Payments Solution.
+
+```mermaid
+classDiagram
+  class MyPaymentService {
+    + sendPaymentViaGevamu()
+  }
+  class GevamuFacade {
+    + sendPayment()
+  }
+
+  MyPaymentService ..> GevamuFacade
+```
+
+## Program implementation
+
+Sending payment is quite simple operation. It includes only call of `PaymentFlow`. 
+
+But before starting the flow you should provide payment instruction. It should be created according to chosen payment standard.
+
+```kotlin
+class MyPaymentService(private val gevamuFacade: GevamuFacade) {
+    fun sendPaymentViaGevamu() {
+        // Create payment instruction with payment details
+        val paymentInstruction = PaymentInstruction(/* payment details */)
+        
+        // Specify gateway node for payment processing
+        val gateway = Party(/* gateway node data */)
+        
+        // Use GevamuFacade to send payment instruction to gateway node
+        gevamuFacade.sendPayment(paymentInstruction, gateway)
+    }
+}
+
+class GevamuFacade(private val serviceHub: AppServiceHub): SingletonSerializeAsToken() {
+
+  fun sendPayment(
+    paymentInstruction: PaymentInstruction,
+    gateway: Party
+  ): List<StateAndRef<Payment>> {
+    val flowHandle = serviceHub.startFlow(PaymentFlow(paymentInstruction, gateway, UUID.randomUUID()))
```

Comment:
If I'm not mistaken, the last parameter (UUID) is optional. If it is I'd remove it in this example.

Reply:
That's true, will remove it in the next commit